recipes/ccache-toolchain/activate.sh

```diff
@@ -2,3 +2,11 @@
 
 export _OLD_PATH=${PATH}
 export PATH="${PREFIX}/bin/conda_forge_ccache:${PATH}"
+
+if [ -f "/feedstock_root/build_artefacts" ]; then
+    export CCACHE_DIR=/feedstock_root/build_artefacts/.ccache
+fi
+
+# Set max cache size so we don't carry old objects for too long
+ccache -M 400M
```

Comment:
I think ccache will clear the exceeding size later when you try to compile something, but I'm not sure. We need to check that